webapp/components/test-search.js

```diff
@@ -39,7 +19,11 @@ const QUERY_GRAMMAR = ohm.grammar(`
 
     Exp = NonemptyListOf<OrPart, or>
 
-    OrPart = NonemptyListOf<AndPart, and>
+    OrPart
```

Comment:
Do we need this change? It looks like the non-default alternatives for `OrPart` and `AndPart` are now the same, which is redundant.